examples/using-javascript-transforms/gatsby-node.js

```diff
@@ -70,6 +71,12 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
           reject(result.errors)
         }
 
+        const layoutIndex = path.resolve(`src/layouts/index.js`)
```

Comment:
This isn't necessary. Any component in `src/layouts` automatically becomes a layout. You only use `createLayout` directly if you're creating layouts in other places.